app/views/layouts/_support_links.html.erb

```diff
@@ -1,4 +1,10 @@
-<h2 class="govuk-visually-hidden">Support links</h2>
+<h2 class="govuk-heading-m">Support and guidance</h2>
+<p class="govuk-body-s">
+  If you have a question, or you've had a problem using this service, please contact us at
+  <a href="mailto:continuing-professional-development@digital.education.gov.uk" class="govuk-footer__link">
+    continuing-professional-development@digital.education.gov.uk
+  </a>
+</p>
```

Comment:
Let's use a component for it, `MailToSupportComponent`.

Reply:
Turns out it already exists!

It doesn't seem suitable for use here without changing two out of the three things it does. I've changed to use the config variable and `mail_to` helper, though.

Reply:
Fine, so long as we don't hardcode the email it's probably ok.